src/System.Memory/ref/System.Memory.cs

```diff
@@ -108,6 +104,13 @@ public static class SpanExtensions
 
         public static bool StartsWith<T>(this ReadOnlySpan<T> span, ReadOnlySpan<T> value) where T : struct, IEquatable<T> { throw null; }
         public static bool StartsWith(this ReadOnlySpan<byte> span, ReadOnlySpan<byte> value) { throw null; }
+
+        public static ReadOnlySpan<char> AsSpan(this string text) { throw null; }
+
```

Comment:
Nit: I think it would look better without empty lines between the different AsSpan overloads.